### PR TITLE
Introduction to rails

### PR DESCRIPTION
**Pull request by keyur-patel-21**
made new controller "Home" and rooted it to index.html.erb

### CODE REVIEW

File: app/controllers/employees_controller.rb [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+        format.json { render :show, status: :created, location: @employee }
+      else
+        format.html { render :new, status: :unprocessable_entity }
+        format.json { render json: @employee.errors, status: :unprocessable_entity }
+      end
+    end
+  end
+
+  # PATCH/PUT /employees/1 or /employees/1.json
+  def update
```
Comment (DhrutiPatelG) [inline app/controllers/employees_controller.rb:38]: defined update method correctly. find(params[:id]) is missing

File: app/controllers/employees_controller.rb [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+        format.json { render :show, status: :ok, location: @employee }
+      else
+        format.html { render :edit, status: :unprocessable_entity }
+        format.json { render json: @employee.errors, status: :unprocessable_entity }
+      end
+    end
+  end
+
+  # DELETE /employees/1 or /employees/1.json
+  def destroy
```
Comment (DhrutiPatelG) [inline app/controllers/employees_controller.rb:51]: find(params[:id]) is missing

File: app/views/employees/_form.html.erb [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+  <div class="field" "form-group">
+    
+    <%= form.text_field :university, class:"form-control", placeholder:"University"%>
+  </div>
+
+  <br>
+  <div class="actions">
+    <%= form.submit class:"btn btn-dark" %>
+  </div>
+<% end %>
```
Comment (DhrutiPatelG) [inline app/views/employees/_form.html.erb:38]: remove extra lines in div tags

File: app/views/employees/show.html.erb [hunk 4]
```diff
@@ -0,0 +1,13 @@
+
+<div class="jumbotron">
+<h1 class="display-4"><%= @employee.first_name %><%= @employee.last_name %></h1>
```
Comment (DhrutiPatelG) [inline app/views/employees/show.html.erb]: add tabs wherever require

File: app/views/employees/show.html.erb [hunk 5]
```diff
@@ -0,0 +1,13 @@
...
+<p class="lead"><%= @employee.position %></p>
+<hr class="my-4">
+<p><%= @employee.department %> | <%= @employee.university %></p>
+
+</div>
+
+
```
Comment (DhrutiPatelG) [inline app/views/employees/show.html.erb]: remove extra blank line

File: app/views/home/_header.html.erb [hunk 6]
```diff
@@ -0,0 +1,33 @@
...
+    
+    <button class="navbar-toggler" type="button" data-bs-toggle="collapse" data-bs-target="#navbarSupportedContent" aria-controls="navbarSupportedContent" aria-expanded="false" aria-label="Toggle navigation">
+      <span class="navbar-toggler-icon"></span>
+    </button>
+    <div class="collapse navbar-collapse" id="navbarSupportedContent">
+      <ul class="navbar-nav me-auto mb-2 mb-lg-0">
+        
+        <li class="nav-item">
+          <%= link_to "About us", home_about_path, class:"nav-link" %>
+
```
Comment (DhrutiPatelG) [inline app/views/home/_header.html.erb]: remove extra blank lines

File: app/controllers/home_controller.rb [hunk 7]
```diff
@@ -0,0 +1,8 @@
+class HomeController < ApplicationController
+  def index
+  end
+
+  def about
+  end
+  
+end
```
Comment (DhrutiPatelG) [inline app/controllers/home_controller.rb:8]: check naming convention for controller

File: config/routes.rb [hunk 8]
```diff
@@ -1,3 +1,7 @@
 Rails.application.routes.draw do
+  resources :employees
+  # get 'home/index'
+  root 'home#index'
+  get "home/about"
```
Comment (DhrutiPatelG) [inline config/routes.rb:5]: add proper routing

File: app/views/employees/show.html.erb (hunk 4 above)
Comment (keyur-patel-21) [reply app/views/employees/show.html.erb]: Done

**Pull request merged**